### PR TITLE
LaRoSeDa

### PR DESCRIPTION
**Pull request by MihaelaGaman**
Add LaRoSeDa to huggingface datasets.

### CODE REVIEW

File: datasets/laroseda/README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/ancatache/LaRoSeDa
+- **Repository:** https://github.com/ancatache/LaRoSeDa
+- **Paper:** Anca Maria Tache, Mihaela Gaman, Radu Tudor Ionescu. Clustering Word Embeddings with Self-Organizing Maps. Application on LaRoSeDa – A Large Romanian Sentiment Data Set. ArXiv, 2021 [(link to paper)](https://arxiv.org/pdf/2101.04197.pdf).
+- **Leaderboard:** [Needs More Information]
+- **Point of Contact:** raducu.ionescu@gmail.com
```
Comment (lhoestq) [inline datasets/laroseda/README.md:54]:
```suggestion
- **Homepage:** [Github](https://github.com/ancatache/LaRoSeDa)
- **Repository:** [Github](https://github.com/ancatache/LaRoSeDa)
- **Paper:** [Arxiv](https://arxiv.org/abs/2101.04197)
- **Leaderboard:** [Needs More Information]
- **Point of Contact:** raducu.ionescu@gmail.com
```

File: datasets/laroseda/README.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+### Citation Information
+
+@article{
+    tache2101clustering,
+    title={Clustering Word Embeddings with Self-Organizing Maps. Application on LaRoSeDa -- A Large Romanian Sentiment Data Set},
+    author={Anca Maria Tache and Mihaela Gaman and Radu Tudor Ionescu},
+    journal={ArXiv},
+    year = {2021}
+}
```
Comment (lhoestq) [inline datasets/laroseda/README.md:167]:
````suggestion
```
@article{
    tache2101clustering,
    title={Clustering Word Embeddings with Self-Organizing Maps. Application on LaRoSeDa -- A Large Romanian Sentiment Data Set},
    author={Anca Maria Tache and Mihaela Gaman and Radu Tudor Ionescu},
    journal={ArXiv},
    year = {2021}
}
```
````
add code block

File: datasets/laroseda/laroseda.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    journal={ArXiv},
+    year = {2021}
+}
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+        LaRoSeDa (A Large Romanian Sentiment Data Set) contains 15,000 reviews written in Romanian, of which 7,500 are positive and 7,500 negative.
+        Star ratings of 1 and 2 and of 4 and 5 are provided for negative and positive reviews respectively.
+        The current dataset uses star rating as the label for multi-class clasification.
```
Comment (lhoestq) [inline datasets/laroseda/laroseda.py:39]:
```suggestion
LaRoSeDa (A Large Romanian Sentiment Data Set) contains 15,000 reviews written in Romanian, of which 7,500 are positive and 7,500 negative.
Star ratings of 1 and 2 and of 4 and 5 are provided for negative and positive reviews respectively.
The current dataset uses star rating as the label for multi-class classification.
```
indentation + typo

File: datasets/laroseda/laroseda.py [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+
+class LaRoSeDaConfig(datasets.BuilderConfig):
+    """BuilderConfig for the LaRoSeDa dataset"""
+
+    def __init__(self, **kwargs):
+        super(LaRoSeDaConfig, self).__init__(**kwargs)
+
+
+class LaRoSeDa(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/laroseda/laroseda.py:61]:
```suggestion
class Laroseda(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `laroseda`
You'll need to regenerate the dataset_infos.json file after this change

**Review by lhoestq — COMMENTED** [review]
Cool thank you !
Good job on the dataset card as well :)

**Comment by MihaelaGaman** [comment]
@lhoestq all the changes requested are implemented. Thank you for your time and feedback :)

**Review by lhoestq — APPROVED** [review]
Awesome thank you ! :)

**Pull request merged**